akka-docs/rst/java/code/docs/http/javadsl/server/directives/FileAndResourceDirectivesExamplesTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2016-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package docs.http.javadsl.server.directives;
+
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.StatusCodes;
+import akka.http.javadsl.server.PathMatchers;
+import akka.http.javadsl.server.Route;
+import akka.http.javadsl.server.directives.DirectoryRenderer;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+import org.junit.Ignore;
+import org.junit.Test;
+import scala.NotImplementedError;
+
+import static akka.http.javadsl.server.PathMatchers.segment;
+
+public class FileAndResourceDirectivesExamplesTest extends JUnitRouteTest {
+
+  @Ignore("Compile only test")
+  @Test
+  public void testGetFromFile() {
+    //#getFromFile
+    final Route route = path(PathMatchers.segment("logs").slash(segment()), name ->
+      getFromFile(".log")
```

Comment:
Since it is just a compile-only test, it doesn't matter what the file name is. However, your guess makes more sense. So, am I supposed to correct it both in this sample and the original sample?